style.css

```diff
@@ -35,10 +35,64 @@ header {
   align-content: center;
 }
 
-.navbarMenue {
-  display: none;
+/* mobile menu section */
+@media only screen and (max-width: 768px) {
+  .navbarMenue {
+    visibility: hidden;
+    opacity: 0;
+    width: 100%;
+    height: 100vh;
+    background: rgba(96, 112, 255, 0.95);
+    filter: contrast(1.75);
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+    top: 0;
+    margin: 0;
+    padding-left: 20px;
+    left: 100%;
+    transition: left 0.8s, visibility 0.8s, opacity 0.4s;
+    z-index: 999;
+  }
+
+  .navbarMenuHeader {
+    display: flex;
+    justify-content: end;
+    padding-right: 38px;
+    padding-top: 40px;
+    padding-bottom: 32px;
+  }
```

Comment:
- [ ] As suggested by the previous reviewer, I would as recommend you add a backdrop to blur your background. You can read more on how to use backdrop-filter [here](https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter)